code/app/src/main/java/com/adobe/marketing/mobile/messagingsample/MainActivity.kt

```diff
@@ -31,6 +32,11 @@ class MainActivity : AppCompatActivity() {
         btn_getLocalNotification.setOnClickListener {
             scheduleNotification(getNotification("Click on the notification for tracking"), 1000)
         }
+        btn_triggerIAM.setOnClickListener {
```

Comment:
Since Kotlin Synthetic is used, I would keep the id of views in Camel case. e.g. Here use `btnTriggerIAM` instead of `btn_triggerIAM`. That looks more Kotlin/Java type.